bench/workgen/workgen.cxx

```diff
@@ -162,6 +163,38 @@ thread_idle_table_cycle_workload(void *arg)
     return (nullptr);
 }
 
+static void *
+thread_dynamic_create_workload(void *arg)
+{
+    WorkloadRunnerConnection *runnerConnection = static_cast<WorkloadRunnerConnection *>(arg);
+    WT_CONNECTION *connection = runnerConnection->connection;
+    WorkloadRunner *runner = runnerConnection->runner;
+
+    try {
+        runner->start_dynamic_create(connection);
+    } catch (WorkgenException &wge) {
+        std::cerr << "Exception while dynamically creating tables." << std::endl;
```

Comment:
I think the `WorkgenException` can contain a message (`wge._str`). Do we want to print it here?

Reply:
I was being consistent here with other threads throwing errors. I tested and returning ```wge._str``` gives more useful information which was otherwise not coming through. I have fixed this in other threads too.

Reply:
Nice! Thanks!